### PR TITLE
Update header menus markup and styles

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
A review please.

Update auth service UI 
screen shot 2018-06-06 at 16 19 20

screen shot 2018-06-05 at 10 17 09

**Comment by sewagodimo** [comment]
is this an update for both springster and Zathu or did you change Zathu to springster?

File: gem/styles/gem-malawi/auth/zathu.enhanced.scss [hunk 1]
```diff
@@ -284,28 +286,32 @@ hr {
...
-
     h1 {
       color: #fff;
       font-size: 1.2em;
     }
   }
-
   &-list {
-    padding-top: $spacer-half;
+    padding-top: 0;
```
Comment (sewagodimo) [inline gem/styles/gem-malawi/auth/zathu.enhanced.scss]: is this not the same as not having padding?

File: gem/styles/gem-malawi/auth/zathu.feature.scss [hunk 2]
```diff
@@ -232,23 +256,47 @@ hr {
   }
 }
 
+.Footer {
+  background: $colorGrey;
+  border: 0;
```
Comment (sewagodimo) [inline gem/styles/gem-malawi/auth/zathu.feature.scss]: why are some values set to zero?

File: gem/styles/gem-malawi/auth/zathu.enhanced.scss (hunk 1 above)
Comment (Mitso) [reply gem/styles/gem-malawi/auth/zathu.enhanced.scss]: No quite. For Zathu yes, however we are overriding GE default styles inheritance elsewhere with every rule.

File: gem/styles/gem-springster/auth/springster.feature.scss [hunk 3]
```diff
@@ -61,40 +61,48 @@ hr {
...
   &-branding {
     padding: $spacer-mid;
-
     &-image {
       background: url('/static/authentication_service/images/sp-logo.png') no-repeat;
       background-position: 50%;
       background-size: contain;
       display: block;
-      height: 60px;
+      height: 35px;
```
[the remark this answers is not in the document]
Comment (Mitso) [reply gem/styles/gem-springster/auth/springster.feature.scss]: Better to use pixels for images sizes which is relative to the viewing device.

**Pull request merged**